examples/pyspark_ml_autologging/pipeline.py

```diff
@@ -0,0 +1,36 @@
+from pyspark.ml.classification import LogisticRegression
+from pyspark.ml.feature import VectorAssembler, StandardScaler
+from pyspark.ml import Pipeline
+from pyspark.sql import SparkSession
+from sklearn.datasets import load_iris
+
+import mlflow
+
+spark = SparkSession.builder.getOrCreate()
+mlflow.pyspark.ml.autolog()
+
+df = load_iris(as_frame=True).frame.rename(columns={"target": "label"})
+df = spark.createDataFrame(df)
+train, test = df.randomSplit([0.8, 0.2])
+
+assembler = VectorAssembler(inputCols=df.columns[:-1], outputCol="features")
+scaler = StandardScaler(inputCol=assembler.getOutputCol(), outputCol="scaledFeatures")
+lor = LogisticRegression(maxIter=5, featuresCol=scaler.getOutputCol())
+
+# Non-neseted pipeline
+pipeline = Pipeline(stages=[assembler, scaler, lor])
+with mlflow.start_run():
+    pipelineModel = pipeline.fit(train)
+
+pred = pipelineModel.transform(test)
+pred.select(pipeline.getStages()[-1].getPredictionCol()).show(10)
+
+# Nested pipeline
+nestedPipeline = Pipeline(stages=[Pipeline(stages=[assembler, scaler]), lor])
+with mlflow.start_run():
+    nestedPipelineModel = nestedPipeline.fit(train)
+
+pred = nestedPipelineModel.transform(test)
+pred.select(nestedPipeline.getStages()[-1].getPredictionCol()).show(10)
```

Comment:
Why not display both "featuers" column and prediction column

Reply:
got it, let's just add the features column.